src/components/vr-scatter-plot/vr-scatter-plot.component.scss

```diff
@@ -0,0 +1,37 @@
+@import "../../utils/constants";
+
+:host {
+  flex: 1;
```

Comment:
nit: Please alphabetize through out the file

Reply:
what do you mean by nit?